src/style.css

```diff
@@ -1,3 +1,79 @@
+* {
+  margin: 0;
+  padding: 0;
+  color: rgba(0, 0, 0, 1);
+  font-family: Roboto, serif;
```

Comment:
Consider adding a fallback font after 'Roboto' to ensure text is displayed correctly if 'Roboto' is unavailable. For example: `font-family: Roboto, Arial, sans-serif;`